2022: renamed updateManuscript mutation to updateSubmission

erezmus opened the pull request.

erezmus committed:
For #2022: renamed updateManuscript mutation to updateSubmission

erezmus committed:
restored submitManuscript in disclosure step

will-byrne committed:
merge develop into branch

will-byrne committed:
merge in develop and alter new submission wizzard to match branch

diversemix commented on packages/component-submission/client/graphql/withGQL.js:
```diff
@@ -17,7 +17,7 @@ export default compose(
   }),
   graphql(UPDATE_MANUSCRIPT, {
-    name: 'updateManuscript',
+    name: 'updateSubmission',
```
Where does `UPDATE_MANUSCRIPT` come from on the line above?

Cooryd replied:
Good catch, this import doesn't exist anymore, needs changing on line 6

Commit:
Re-name mutation being imported into withGQL

old UPDATE_MANUSCRIPT mutation is now named updateSubmission so the import was no longer valid

diversemix commented:
Looks good @Cooryd - let's get it in?
